app/assets/locales/android_translatable_strings.txt

```diff
@@ -381,6 +381,15 @@ notification.sync.connections.title=No Connection
 notification.sync.connections.detail=No active data connections
 notification.sync.connections.action=You don't have any active data connections configured. Try connecting your phone to wifi or turning mobile data on.
 
+
+connection.captive_portal.title=Captive Portal Detected

```

Comment:
Our users won't know what a captive portal is. We should just say that No Internet Available and then explain in details that they are connected to a wifi they need to sign in into.